repair/row_level.cc

```diff
@@ -1118,6 +1146,63 @@ class repair_meta {
             // We are about to create a real evictable reader, so drop the fake
             // reader (evicted or not), we don't need it anymore.
             _db.local().get_reader_concurrency_semaphore().unregister_inactive_read(std::move(_fake_inactive_read_handle));
+            if (is_incremental_repair()) {
```

Comment:
Could we share the code paths between incremental and full? full would just merge the two LSMs before initiating repair.

Reply:
This is how we create the reader for incremental repair. The full repair reader path is much more complicated, which involves local reader, multishard split reader, multishard filter reader. It makes no sense to combine them. In addition, we could implement full repair by reset the sstables_reapried_at in system.tablets to zero so all sstables are marked as unrepaired. This way we already share the code for both incremental and full repair.